bullet_train-themes/app/views/themes/base/fields/_ckeditor.html.erb

```diff
@@ -9,9 +9,9 @@ stimulus_controller = 'fields--ckeditor'
 
 form ||= current_fields_form
 options ||= {}
-options = options.merge({ data: {"#{stimulus_controller}-target": 'field' }})
 other_options ||= {}
 
+options = options.merge({ data: {"#{stimulus_controller}-target": 'field' }})
```

Comment:
This file seems like a remnant of a feature that we no longer support. Maybe we should remove it in the near future?

Reply:
Just added a TODO in https://github.com/bullet-train-co/bullet_train-core/pull/633/commits/660bc41d3b40b23074b9b0de11ebcd2a3c439e73 to remind as later (I don't have a proper solution off the top of my head).